administrator/components/com_workflow/View/Workflow/HtmlView.php

```diff
@@ -140,7 +140,7 @@ protected function addToolbar()
 
 			if ($itemEditable && !$this->item->core)
 			{
-				//ToolbarHelper::apply('workflow.apply');
+				// ToolbarHelper::apply('workflow.apply');
```

Comment:
dont know why this code has been commented out but if its not needed then it should be deleted otherwise the comment shoukd be removed